### PR TITLE
Add collations page

### LINKED ISSUE #21: Intl.Locale Info API
Create Documentation for ** Intl.Locale Info API** 

- [x] Review Readme [documentation and examples](https://github.com/tc39/proposal-intl-locale-info)
- [x] Create/Update MDN Main Docs Page - [Page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale)

**MDN Pages :** 
- PR https://github.com/mdn/content/pull/8985
- [x] Intl.Locale.prototype.weekInfo
- [x] Intl.Locale.prototype.timeZones 
- [x] Intl.Locale.prototype.hourCycles 
- [x] Intl.Locale.prototype.numberingSystems
- [x] Intl.Locale.prototype.textInfo 
- PR https://github.com/mdn/content/pull/8768
 - [x] Intl.Locale.prototype.calendars 
 - [x]  Intl.Locale.prototype.collations

 **Browser compat-data :**  PR https://github.com/mdn/browser-compat-data/pull/12314
- [x] Intl.Locale.prototype.weekInfo
- [x] Intl.Locale.prototype.timeZones
- [x] Intl.Locale.prototype.hourCycles
- [x] Intl.Locale.prototype.numberingSystems
- [x] Intl.Locale.prototype.calendars
- [x] Intl.Locale.prototype.textInfo

### PR DESCRIPTION
**Pull request by romulocintra**
### Description
Add missing page about "collations" for Intl.Locale 

### Related issues and pull requests
 Fixes https://github.com/tc39/ecma402-mdn/issues/21

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/intl/locale/collations/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  - Locale
+  - Localization
+  - Prototype
+  - Reference
+browser-compat: javascript.builtins.Intl.Locale.collations
+---
+
+{{JSRef}}
+
+The **`Intl.Locale.prototype.collations`** property is an accessor property that returns an array of one or more  [collation types](https://www.unicode.org/reports/tr35/tr35-collation.html#CLDR_collation) for the `Locale`.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/intl/locale/collations/index.md:18]:
```suggestion
The **`Intl.Locale.prototype.collations`** property is an accessor property that returns an array of one or more [collation types](https://www.unicode.org/reports/tr35/tr35-collation.html#CLDR_collation) for the `Locale`.
```

File: files/en-us/web/javascript/reference/global_objects/intl/locale/collations/index.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+## Description
+
+Collation is the process of ordering strings of characters. It is used whenever strings must be sorted and placed into a certain order, from search query results to ordering records in a database. While the idea of placing strings in order might seem trivial, the idea of order can vary from region to region and language to language. The array of `collations`  returned helps to make it easier for JavaScript programmers to access the collation types used by a particular locale.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/intl/locale/collations/index.md:22]:
```suggestion
Collation is the process of ordering strings of characters. It is used whenever strings must be sorted and placed into a certain order, from search query results to ordering records in a database. While the idea of placing strings in order might seem trivial, the idea of order can vary from region to region and language to language. The array returned from `collations` helps to make it easier for JavaScript programmers to access the collation types used by a particular locale.
```

**Review by Josh-Cena — COMMENTED** [review]
There's some duplication from `Intl.Locale#collation`. Is it possible that we redirect the list of types to the `collation` page instead?

I'm curious whether `collations` is ordered in any way. Does the value returned from `collation` correspond to the first value in `collations`?

File: files/en-us/web/javascript/reference/global_objects/intl/locale/collations/index.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+The {{jsxref("Intl/Locale/Locale", "Intl.Locale()")}} constructor has an optional configuration object argument, which can contain any of several extension types, including collations types. Set the `collations` property of the configuration object to your desired collations type, and then pass it into the constructor.
+
+```js
+const locale = new Intl.Locale("zh-Hant", { collations: "zhuyin" });
+console.log(locale.collations); // Prints ["zhuyin"]
+```
+
+### Obtaining supported collation types
+
+List supported `collation types` for a given `Locale`.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/intl/locale/collations/index.md:97]:
```suggestion
List supported `collation` types for a given `Locale`.
```

**Comment by Josh-Cena** [comment]
I see in the spec draft:

> sorted in descending preference of those in common use for string comparison in locale. The values "standard" and "search" must be excluded from list.

This information seems useful.

**Comment by Josh-Cena** [comment]
Pinging @romulocintra this is waiting for the reviews above to be addressed

**Comment by romulocintra** [comment]
> I see in the spec draft:
> 
> > sorted in descending preference of those in common use for string comparison in locale. The values "standard" and "search" must be excluded from list.
> 
> This information seems useful.

Hi @Josh-Cena have to look at your comments, but no bandwidth so far and apologies for that. Can we land this PR cause will provide basic documentation and then incrementally improve it?

**Review by Josh-Cena — APPROVED** [review]
Actually I'm going to merge it anyway, because we have the same duplication and underexplanation issue elsewhere, and it will take a separate PR to fix them all. Thanks!

**Pull request merged**